library/std/Cargo.toml

```diff
@@ -27,10 +27,10 @@ addr2line = { version = "0.19.0", optional = true, default-features = false }
 rustc-demangle = { version = "0.1.21", features = ['rustc-dep-of-std'] }
 miniz_oxide = { version = "0.6.0", optional = true, default-features = false }
 [dependencies.object]
-version = "0.30.0"
+version = "0.31.0"
```

Comment:
Does it build without this change? `backtrace-rs` doesn't have xcoff support yet anyway.